fplll/test.cpp

```diff
@@ -19,17 +19,47 @@
 using namespace std;
 using namespace fplll;
 
+
```

Comment:
Lines 55 and 56 need to be dropped as well: https://github.com/fplll/fplll/blob/master/fplll/Makefile.am#L55